plottable.js

```diff
@@ -6425,16 +6425,32 @@ var Plottable;
                 }
                 return this;
             };
-            AbstractPlot.prototype.removeDataset = function (datasetOrKeyOrArray) {
+            /**
+             * Removes a dataset by the given identifier
+             *
+             * @param {string | Dataset | any[]} datasetIdentifer The identifier for the dataset to remove
+             * If a string is inputted, it is interpreted as the dataset key to remove.
+             * If a Dataset is inputted, the first Dataset in the plot that is the same will be removed.
+             * If an any[] is inputted, the first data array in the plot that is the same will be removed.
```

Comment:
Generally, maybe "If ____, [...]" is clearer than "If a ______ is inputted, [...]"